.github/workflows/swift-toolchain.yml

```diff
@@ -203,6 +247,7 @@ jobs:
           echo compilers_build_runner=${{ vars.COMPILERS_BUILD_RUNNER || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
 
       - uses: actions/upload-artifact@v4
+        if: inputs.publish_artifacts != true
```

Comment:
This isn't really an artifact, it's merely tracking state for known valid builds and is what powers the `repo sync -s` to get a "stable" sync point.

Reply:
Ah, that's right. This isn't actually necessary either, since we guard against `git push`ing this file in the snapshot job below. Thanks for catching. Removed.